src/Adapter/Configuration.php

```diff
@@ -124,21 +140,25 @@ public function get($key, $default = null)
      *
      * @param string $key
      * @param mixed $value
+     * @param ShopConstraint|null $shopConstraint
      * @param array $options Options
      *
      * @return $this
      *
      * @throws \Exception
      */
-    public function set($key, $value, array $options = [])
+    public function set($key, $value, ShopConstraint $shopConstraint = null, array $options = [])
```

Comment:
This is a potential BC break, I think we can assume it as I didn't any use of this `$options` parameter in the core code And it's present in none of our interfaces

Reply:
It's a BC Break (you change signature of a public method). Please document it in PR.

Reply:
It is documented

Reply:
Ok. I close